scripts/docker-test.sh

```diff
@@ -1,19 +1,3 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
```

Comment:
Can it be deleted this file since it's empty?

Reply:
:man_facepalming: yes, thanks